src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -315,4 +315,5 @@ public net.minecraft.world.gen.layer.LayerUtil func_202829_a(JLnet/minecraft/wor
 private-f net.minecraft.world.raid.Raid$WaveMember field_221284_f # VALUES
 public net.minecraft.world.server.ServerChunkProvider field_186029_c # chunkGenerator
 public net.minecraft.world.server.ServerChunkProvider field_73251_h # worldObj
-public net.minecraft.world.storage.FolderName <init>(Ljava/lang/String;)V # constructor
\ No newline at end of file
+public net.minecraft.world.storage.FolderName <init>(Ljava/lang/String;)V # constructor
+public-f net.minecraft.entity.EntityClassification field_233667_g_ # CODEC
```

Comment:
For future reference (or if I am wrong about modifying the codec being wrong), when you modify the access transformer file you should run the `checkATs` task as it orders things alphabetically and `entity` should be way before `world`.